src/txn/txn_rollback_to_stable.c

```diff
@@ -317,6 +317,7 @@ __rollback_row_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW
         WT_ERR(__wt_upd_alloc_tombstone(session, &hs_upd, NULL));
         WT_ERR(__wt_hs_modify(cbt, hs_upd));
         WT_STAT_CONN_INCR(session, txn_rts_hs_removed);
+        WT_STAT_CONN_INCR(session, cache_hs_key_truncate_rts_unstable);
```

Comment:
I don't like how this stat works since it increases per record removed (as opposed to the number of times we truncate a range of keys) which isn't how most of the existing stats work (unlike the `call` variant).

At the very least, we should make this clear in the statistic name.

Reply:
@tetsuo-cpp. Agreed. I have changed the description for the stat in 31703d8.